go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cloudflare/terraform-provider-cloudflare
 go 1.15
 
 require (
-	github.com/cloudflare/cloudflare-go v0.19.0
+	github.com/cloudflare/cloudflare-go v0.19.1-0.20210728051236-7210b1583143
```

Comment:
can you please remove this from your PR? we bump this separately to the PR.

Reply:
Will do 👍